s2tbx-s2msi-reader/src/main/java/org/esa/s2tbx/dataio/s2/ortho/Sentinel2OrthoProductReader.java

```diff
@@ -315,20 +312,19 @@ protected final Product readProduct(String defaultProductName, boolean isGranule
                     || anglesGridsMap.size() > 0) {
                 addAnglesBands(mapCRS, defaultProductSize, product, sceneDescription, anglesGridsMap,
                         productDefaultGeoCoding, subsetDef);
-            } else {
+//            } else {
                 // todo: add reader L2HF for the angle tif data
             }
         }
 
         for (S2Metadata.Tile tile : tileList)
-            addGRIBBand(product, tile, sceneDescription, mapCRS, namingConvention);
+            addGRIBBand(product, tile, namingConvention);
 
         return product;
     }
 
-    private void addGRIBBand(Product product, S2Metadata.Tile tile, S2OrthoSceneLayout sceneDescription,
-            CoordinateReferenceSystem mapCRS, INamingConvention namingConvention)
-            throws IOException, NoSuchAuthorityCodeException, FactoryException {
+    private void addGRIBBand(Product product, S2Metadata.Tile tile, INamingConvention namingConvention)
+            throws IOException {
         VirtualPath tileFolder = namingConvention.findGranuleFolderFromTileId(tile.getId());
         S2Metadata.ProductCharacteristics characteristicsAUXDATA = new S2Metadata.ProductCharacteristics();
```

Comment:
On characteristicsAUXDATA values are set but they are not used afterwards. Can probably be removed.